templates/store/publisher-details.html

```diff
@@ -0,0 +1,226 @@
+{% extends webapp_config['LAYOUT'] %}
+
+{% block meta_copydoc %}{% endblock %}
+
+{# TODO: title, meta, etc #}
+{% block meta_title %}{{ publisher }} in the Snap Store | Snapcraft{% endblock %}
+{% block meta_description %}{{ publisher }} publishes snaps for Linux in the Snap Store.{% endblock %}
```

Comment:
Probably we can update the title to include: `Install {{ publisher }} software for Linux in the Snap Store`

Reply:
@therealjuan You mean description?